src/components/Sidebar.tsx

```diff
@@ -1,58 +1,118 @@
 import Link from "next/link";
 import Image from "next/legacy/image";
 import { signOut } from "next-auth/react";
+import {
+  faHouse,
+  faUsers,
+  faUserGroup,
+  faUserPlus,
+  faCircleUser,
+  faCity,
+  faArrowRightFromBracket,
+  IconDefinition,
+} from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 
-const SidebarItem = ({ label } : { label: string }) => {
+interface Button {
+  name: string;
+  color: string;
+  icon: IconDefinition;
+}
+
+const userLinks: Button[] = [
+  {
+    name: "Home",
+    color: "#000000",
+    icon: faHouse,
+  },
+  {
+    name: "Students",
+    color: "#000000",
+    icon: faUsers,
+  },
+  {
+    name: "Seniors",
+    color: "#000000",
+    icon: faUserGroup,
+  },
+  {
+    name: "Pending",
+    color: "#000000",
+    icon: faUserPlus,
+  },
+  {
+    name: "Profile",
+    color: "#000000",
+    icon: faCircleUser,
+  },
+];
+
+const SidebarItem = ({
+  label,
+  iconName,
+}: {
+  label: string;
+  iconName: IconDefinition;
+}) => {
   return (
-    <div className="w-full px-4 p-2 text-md text-left font-serif duration-150 hover:bg-darker-tan">
-      <div className="duration-150 hover:translate-x-1">{label}</div>
+    <div className="text-md flex w-full space-x-4 p-2 px-10 text-left font-serif duration-150 hover:translate-x-1">
+      <div className="w-1/6 duration-150 hover:translate-x-1">
```

Comment:
Assumes you did the padding below, you can remove `w-1/6`, `px-10` here. As an aside, `p-2`  overlaps with `px-10`. I think it's easier to parse code if it's `py-<some number` and `px-10`.